src/dmd/toir.d

```diff
@@ -490,15 +490,12 @@ int intrinsic_op(FuncDeclaration fd)
 Lva_start:
     if (global.params.is64bit &&
         fd.toParent().isTemplateInstance() &&
-        fd.ident == Id.va_start)
+        id3 == Id.va_start &&
+        id2 == Id.stdarg &&
+        id1 == Id.stdc &&
+        (*md.packages)[1] == Id.core)
```

Comment:
packages are not in reverse order, so this should be `id1 == Id.core && (*md.packages)[1] == Id.stdc`.

Reply:
Ah, thanks!